pom.xml

```diff
@@ -132,4 +132,4 @@
         </dependency>
     </dependencies>
 
-</project>
+</project>
```

Comment:
Nitpick but no trailing newline? Why touch this in the first place? It's completely unrelated to your main change.